### PR TITLE
Replace prohobited terms

### PR DESCRIPTION
**Pull request by anjumfatima90**
for issue https://github.com/OpenLiberty/open-liberty/issues/17254

### CODE REVIEW

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by mbroz2** [comment]
#build

**Review by mbroz2 — APPROVED** [review]
@pnicolucci @volosied does jsf-2.2 still see updates?   I couldn't find instances of this in 2.3 or beyond so it seems it's been resolved upstream in new versions.

**Comment by anjumfatima90** [comment]
#build

**Comment by volosied** [comment]
Hi, @mbroz2,  I addressed the terms in this PR via https://github.com/OpenLiberty/open-liberty/pull/20993 

As for  jsf-2.0 and jsf-2.2, we have a copy of the source code in CL/OL, respectfully.  As for 2.3 and 3.0, we pull in the code via the jars (uploaded to artifactory).  

JSF should all comply now with the terms in Liberty and in the open source ( https://issues.apache.org/jira/browse/MYFACES-4435 )

File: dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/conf/authn/remoteuser-internal-authn-config.xml [hunk 1]
```diff
@@ -41,7 +41,7 @@
         -->
     </util:list>
     
-    <!-- Uncomment/configure to install username whitelist, blacklist, and/or match expressions. -->
+    <!-- Uncomment/configure to install username, and/or match expressions. -->
```
Comment (mbroz2) [inline dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/conf/authn/remoteuser-internal-authn-config.xml]:
```suggestion
    <!-- Uncomment/configure to install username restrictions (allowing/blocking lists), and/or match expressions. -->
```

File: dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/conf/global.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <!-- Use this file to define any custom beans needed globally. -->
 
     <!--
-    Algorithm whitelists and blacklists that override or merge with library defaults. Normally you can leave
+    Algorithm that override or merge with library defaults. Normally you can leave
```
Comment (mbroz2) [inline dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/conf/global.xml]:
```suggestion
    Algorithm restrictions (allowing/blocking lists) that override or merge with library defaults. Normally you can leave
```

File: dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/conf/intercept/consent-intercept-config.xml [hunk 3]
```diff
@@ -44,7 +44,7 @@
     <!-- Attribute Release configuration -->
 
     <!--
-    Attribute release whitelist, blacklist, and match expressions to determine 
+    Attribute release attribute IDs, and match expressions to determine
```
Comment (mbroz2) [inline dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/conf/intercept/consent-intercept-config.xml]:
```suggestion
    Attribute release attribute IDs restrictions (allowing/blocking lists), and match expressions to determine 
```

File: dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/dist/conf/authn/remoteuser-internal-authn-config.xml.dist [hunk 4]
```diff
@@ -41,7 +41,7 @@
         -->
     </util:list>
     
-    <!-- Uncomment/configure to install username whitelist, blacklist, and/or match expressions. -->
+    <!-- Uncomment/configure to install username, and/or match expressions. -->
```
Comment (mbroz2) [inline dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/dist/conf/authn/remoteuser-internal-authn-config.xml.dist]:
```suggestion
    <!-- Uncomment/configure to install username restrictions (allowing/blocking lists), and/or match expressions. -->
```

File: dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/dist/conf/global.xml.dist [hunk 5]
```diff
@@ -15,7 +15,7 @@
     <!-- Use this file to define any custom beans needed globally. -->
 
     <!--
-    Algorithm whitelists and blacklists that override or merge with library defaults. Normally you can leave
+    Algorithm that override or merge with library defaults. Normally you can leave
```
Comment (mbroz2) [inline dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/dist/conf/global.xml.dist]:
```suggestion
    Algorithm restrictions (allowing/blocking lists) that override or merge with library defaults. Normally you can leave
```

File: dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/dist/conf/intercept/consent-intercept-config.xml.dist [hunk 6]
```diff
@@ -44,7 +44,7 @@
     <!-- Attribute Release configuration -->
 
     <!--
-    Attribute release whitelist, blacklist, and match expressions to determine 
+    Attribute release attribute IDs, and match expressions to determine
```
Comment (mbroz2) [inline dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/3.3.1/dist/conf/intercept/consent-intercept-config.xml.dist]:
```suggestion
    Attribute release attribute IDs restrictions (allowing/blocking lists), and match expressions to determine 
```

File: dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/4.1.0/conf/authn/remoteuser-internal-authn-config.xml [hunk 7]
```diff
@@ -41,7 +41,7 @@
         -->
     </util:list>
     
-    <!-- Uncomment/configure to install username whitelist, blacklist, and/or match expressions. -->
+    <!-- Uncomment/configure to install username, and/or match expressions. -->
```
Comment (mbroz2) [inline dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/4.1.0/conf/authn/remoteuser-internal-authn-config.xml]:
```suggestion
    <!-- Uncomment/configure to install username restrictions (allowing/blocking lists), and/or match expressions. -->
```

File: dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/4.1.0/conf/global.xml [hunk 8]
```diff
@@ -15,7 +15,7 @@
     <!-- Use this file to define any custom beans needed globally. -->
 
     <!--
-    Algorithm whitelists and blacklists that override or merge with library defaults. Normally you can leave
+    Algorithm that override or merge with library defaults. Normally you can leave
```
Comment (mbroz2) [inline dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/4.1.0/conf/global.xml]:
```suggestion
    Algorithm restrictions (allowing/blocking lists) that override or merge with library defaults. Normally you can leave
```

File: dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/4.1.0/conf/intercept/consent-intercept-config.xml [hunk 9]
```diff
@@ -44,7 +44,7 @@
     <!-- Attribute Release configuration -->
 
     <!--
-    Attribute release whitelist, blacklist, and match expressions to determine 
+    Attribute release attribute Ids, and match expressions to determine
```
Comment (mbroz2) [inline dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/4.1.0/conf/intercept/consent-intercept-config.xml]:
```suggestion
    Attribute release attribute IDs restrictions (allowing/blocking lists), and match expressions to determine 
```

**Review by mbroz2 — CHANGES_REQUESTED** [review]
Some minor tweaks to help ensure we don't loose any context in the comments.

File: dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/4.1.0/conf/intercept/consent-intercept-config.xml (hunk 9 above)
Comment (anjumfatima90) [reply dev/com.ibm.ws.security.saml.sso_fat.common/shibboleth-idp/4.1.0/conf/intercept/consent-intercept-config.xml]: done!

**Review by mbroz2 — APPROVED** [review]

**Comment by anjumfatima90** [comment]
#libby

**Pull request merged**